components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -209,11 +209,30 @@ class PageWallet extends React.Component<Props, State> {
     window.open(externalWallet.accountUrl, '_self', 'noreferrer')
   }
 
+  getAccountActivityLink = () => {
+    const { externalWallet } = this.props.rewardsData
+
+    if (!externalWallet) {
+      return undefined
+    }
+
+    if (externalWallet.status === mojom.WalletStatus.kConnected) {
```

Comment:
I think this could just be:
```ts
return externalWallet.status === mojom.WalletStatus.kConnected ? externalWallet.activityUrl : externalWallet.accountUrl
```

Reply:
Makes sense, fixed.